package.json

```diff
@@ -35,6 +35,22 @@
         "path": "./syntaxes/mysql-injection.json"
       }
     ],
+    "commands": [
+      {
+        "command": "mySql.createDatabase",
+        "title": "%mysql.createDatabase%",
+        "category": "%mysql.createDatabase%",
+        "icon": "images/new-database.svg"
```

Comment:
the color is slightly different from the rest of icons in the toolbar,

Reply:
will fix this.